create-keystone-app/src/index.ts

```diff
@@ -47,32 +47,24 @@ async function normalizeArgs(): Promise<Args> {
   };
 }
 
-const installDeps = async (cwd: string): Promise<'yarn' | 'npm'> => {
+function pkgManagerFromUserAgent(userAgent: string | undefined) {
+  if (!userAgent) return 'npm';
+  const pkgSpec = userAgent.split(' ')[0];
+  const [name, _version] = pkgSpec.split('/');
+  return name ?? 'npm';
+}
```

Comment:
This is an adaptation of https://github.com/vitejs/vite/blob/main/packages/create-vite/src/index.ts#L528-L536